File: abc/abc146/abc146_a.cs
// https://atcoder.jp/contests/abc146/submissions/8605371

using System;

namespace CSharp
{
    class Program
    {
        static void Main()
        {
            var S = Console.ReadLine();

            if (S == "SAT")
            {
                Console.WriteLine(1);
            }
            else if (S == "FRI")
            {
                Console.WriteLine(2);
            }
            else if (S == "THU")
            {
                Console.WriteLine(3);
            }
            else if (S == "WED")
            {
                Console.WriteLine(4);
            }
            else if (S == "TUE")
            {
                Console.WriteLine(5);
            }
            else if (S == "MON")
            {
                Console.WriteLine(6);
            }
            else if (S == "SUN")
            {
                Console.WriteLine(7);
            }
        }
    }
}

File: abc/abc146/abc146_a.fs
// https://atcoder.jp/contests/abc146/submissions/12155271

stdin.ReadLine()
|> fun S ->
    match S with
    | "SUN" -> 7
    | "MON" -> 6
    | "TUE" -> 5
    | "WED" -> 4
    | "THU" -> 3
    | "FRI" -> 2
    | "SAT" -> 1
    | _ -> 0
|> stdout.WriteLine
